issue/src/lib.rs

```diff
@@ -137,6 +165,17 @@ pub fn run(options: Options) -> anyhow::Result<()> {
         } => {
             issues.create(&project, &title, &description, &[])?;
         }
+        Operation::Reaction { id, emoji } => {
+            if let Some(issue) = issues.get(&project, &id)? {
+                let comment_id = term::comment_select(&issue).unwrap();
```

Comment:
Mmh, calling `rad issue react <id> --emoji` panics on an issue that has no comments yet. Is this intended? Otherwise we might want to handle this case more gracefully.

Reply:
Hmm weird, if there are no comments on an issue it should only allow to select the root comment.. What type of errror you get when it panics?

Reply:
I'm so sorry! Accidentally tried with an older commit on your branch. Is working as intended.

Reply:
No problem! Thank you for testing it out!